util/app/discovery/discovery.go

```diff
@@ -165,6 +167,20 @@ func cmpSupports(ctx context.Context, pluginSockFilePath, appPath, repoPath, fil
 		return nil, nil, false
 	}
 
+	cfg, err := cmpClient.CheckPluginConfiguration(ctx, &empty.Empty{})
+	if err != nil {
+		log.WithFields(log.Fields{
+			common.SecurityField:    common.SecurityMedium,
+			common.SecurityCWEField: common.SecurityCWEMissingReleaseOfFileDescriptor,
```

Comment:
Can we drop these fields? I don't think they're needed. I'll put up another PR to remove them from the unrelated lines.

Reply:
do you mean?

```go
cfg, err := cmpClient.CheckPluginConfiguration(ctx, &empty.Empty{})
	if err != nil {
		log.Errorf("error checking plugin configuration %s, %v", fileName, err)
		return nil, nil, false
        }
```

Reply:
Just for curiosity, why do you think they aren't needed?
Thanks!